apps/gp2-frontend/src/working-groups/EditResource.tsx

```diff
@@ -0,0 +1,52 @@
+import { gp2 as gp2Model } from '@asap-hub/model';
+import { ResourceModal } from '@asap-hub/gp2-components';
+import { gp2 as gp2Routing, useRouteParams } from '@asap-hub/routing';
+
+const { workingGroups } = gp2Routing;
+
+const modalInfo = {
+  title: 'Edit Resource',
+  description:
+    'Edit or delete an existing resource that is being shared privately with your group.',
+};
+
+type EditWorkingGroupResponseProps = {
+  workingGroupId: string;
+  workingGroup: gp2Model.WorkingGroupResponse;
```

Comment:
Does the Id need to be separate from the workingGroup. I thought the workingGroup would contain the Id?

Reply:
or just pass the resources

Reply:
Done